fr/basic-concepts/environments-to-organize-workloads-and-users.md

```diff
@@ -3,14 +3,48 @@ title: "Les environnements pour compartimenter charges de travail et utilisateur
 slug: environnements-pour-compartimenter-charges-de-travail-et-utilisateurs
```

Comment:
from the translation, i would have the title to the following:

Utiliser les environnements pour compartimenter les charges de travail et les utilisateurs

Reply:
Agreed.